data/filter/BaseFilterFieldSpec.js

```diff
@@ -151,7 +154,7 @@ export class BaseFilterFieldSpec extends HoistBase {
         return this.values &&
             this.enableValues &&
             this.supportsOperator(op) &&
-            (op === '=' || op === '!=');
+            (op === '=' || op === '!=' || op === 'includes' || op === 'does not include');
```

Comment:
"not includes" to mirror "not like".  It is isn't grammatical but neither is "ends" or "starts"